resources/patches/opensuse/chromium-dma-buf.patch

```diff
@@ -0,0 +1,20 @@
+--- a/ui/gfx/linux/client_native_pixmap_dmabuf.cc
```

Comment:
Would you mind adding a short description for this patch? I can't tell what this does from the changes alone.

Reply:
No problem

Reply:
Ah, I was more interested in knowing what the change does so I could determine if I needed to include it in `linux_rooted` or not (like the user namespace sandbox patch). But since the source doesn't specify, I'll probably leave it in `opensuse` for now.